addons/pos_restaurant/static/src/js/multiprint.js

```diff
@@ -315,12 +316,12 @@ models.Order = models.Order.extend({
     },
     export_as_JSON: function(){
         var json = _super_order.export_as_JSON.apply(this,arguments);
-        json.multiprint_resume = this.saved_resume;
+        json.multiprint_resume = JSON.stringify(this.saved_resume? this.saved_resume: false);
         return json;
     },
     init_from_JSON: function(json){
         _super_order.init_from_JSON.apply(this,arguments);
-        this.saved_resume = json.multiprint_resume;
+        this.saved_resume = JSON.parse(json.multiprint_resume? json.multiprint_resume: false);
```

Comment:
I have some problem here when I launch my pos and have order stored in localstorage before you commit is applied. I think it will be complicated to modify the manage properly values of type object and of type either object or string (In the use case I reproduce, it cannot perform the parse because the json.multiprint_resume is an object and not a string)

Maybe for the 13.0, we should just modify the _save_server function to modify the order.multiprint_resume to a string. And in the function '_get_from_server' parse it from the server like that it won't cause trouble for the local orders. And in the master branch we'll apply this changes in the import/export